circus-train-core/src/main/java/com/hotels/bdp/circustrain/core/PartitionedTableMetadataUpdateReplication.java

```diff
@@ -126,9 +142,7 @@ private PartitionsAndStatistics filterOnReplicatedPartitions(
         replicaClient.getPartition(replicaDatabaseName, replicaTableName, partition.getValues());
         statisticsByPartition.put(partition, sourcePartitionsAndStatistics.getStatisticsForPartition(partition));
       } catch (NoSuchObjectException e) {
-        LOG.debug(
-            "Partition {} not found on replica, skipping metadata update (Set ReplicationMode.FULL to replicate missing partitions and data)",
-            partition.getValues());
+        // partition doesn't exist, skipping it
```

Comment:
A debug message would probably still be useful here

Reply:
done